src/test/java/com/revature/services/JMeterResponseCollectorTest.java

```diff
@@ -21,7 +21,7 @@
 class JMeterResponseCollectorTest {
     private JMeterResponseCollector jmrc;
     private StandardJMeterEngine engine;
-    private static final String JMeterPropPath = "src/test/resources/test.properties";
+    private static final String JMeterPropPath = "src/main/resources/test.properties";
```

Comment:
Was there a need to move a test specific properties file to `src/main` instead of `src/test`?

Reply:
We changed the test.properties to jmeter.properties instead. It is actually a general properties file for jmeter, not a test related config.